### DISCUSSION TITLE
Project progress consultation

### DISCUSSION THREAD

**Discussion by qwt234** [ravynsoft/ravynos#403] [Q&A] [answered]
I have been following this project for a year since August 2022. May I ask how it is currently progressing? Will there be an official version released this year?

I really hope this project can make significant progress.

**Comment by AssassinK786** [comment]
Hi @qwt234 you can check the discord server and git commits 
The OS has changed quite from Aug 2022, @mszoek has managed to a GUI close enough to Catalina but it is far from prefect

**Reply by qwt234** [reply]
Will a new version be released this year? Looking forward to the new version very much. Is it still version 0.4.0?As a Mac enthusiast, I am very much looking forward to the success of this project.

**Reply by AssassinK786** [reply, marked as answer]
@qwt234 
check out this release https://mirrors.nomadlogic.org/nightly/ and try it bare metal

**Reply by qwt234** [reply]
Thank you very much I pay attention to it every day in official website, and I haven't seen a new version. Thank you for sending me the mirror address of the new system version. I will try it.

**Comment by qwt234** [comment]
Excuse me, I downloaded the latest version of ISO and made a startup USB flash drive. But only the desktop and the top menu bar are displayed after starting to enter. There is nothing else. Is this the reason why I am using this one as the configuration of the test computer?Processor:Intel Pentium processor (R) processor G4400@3.30GHz 3.30GHzInstallation memory (RAM):4.00 megabytes (3.89 megabytes available)

**Reply by AssassinK786** [reply]
@qwt234 the work is still in progress as i have said before and it is far from completion

**Reply by qwt234** [reply]
fighting！！Eagerly look forward to your success!

**Comment by mszoek** [comment]
There is a workaround to make the Dock appear on the desktop as well.
```
rm -rf ~/Library/Preferences/*.plist
killall Dock
```

**Reply by qwt234** [reply]
Thank you very much! I wish this project success!
